github/repos_collaborators.go

```diff
@@ -94,7 +94,7 @@ type RepositoryAddCollaboratorOptions struct {
 	Permission string `json:"permission,omitempty"`
 }
 
-// AddCollaborator adds the specified GitHub user as collaborator to the given repo.
+// AddCollaborator invites the specified GitHub user as a collaborator to the given repo.
 //
 // GitHub API docs: https://developer.github.com/v3/repos/collaborators/#add-user-as-a-collaborator
 func (s *RepositoriesService) AddCollaborator(ctx context.Context, owner, repo, user string, opt *RepositoryAddCollaboratorOptions) (*Response, error) {
```

Comment:
I definitely don't think this is a good idea, for 2 reasons. We generally try to prioritize having the method names match what the GitHub API documentation calls the endpoints. In this case:

1. It would be yet a breaking API change, one which is not neccessary, and without strong benefits.
2. https://developer.github.com/v3/repos/collaborators/#add-user-as-a-collaborator calls it "Add user as a collaborator".